moztelemetry/store.py

```diff
@@ -99,7 +99,9 @@ def get_key(self, key):
             value = self.store[key]
         except KeyError:
             raise Exception('Error retrieving key "{}" from S3'.format(key))
-        output = StringIO.StringIO()
+        output = BytesIO()
+        if type(value) is not bytes:
+            value = value.encode('utf-8')
```

Comment:
You can use [`six.ensure_str`](https://github.com/benjaminp/six/blob/68112f3193c7d4bef5ad86ed1b6ed528edd9093d/six.py#L870-L887) here.

Reply:
in this case i'm trying to ensure bytes, in python 3

Reply:
Whoops, that's what I meant as well, tripped over the naming thing heh